app/screens/Charts/Charts.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { useEffect } from 'react';
+import { View, Text } from 'react-native';
+import { useAppDispatch, useAppSelector } from '../../hooks';
+
+import { styles } from './charts.styles';
+
+import {
+  padding,
+  xDomain,
+  yDomain,
+  scatter,
+  stroke,
+  tickCount,
+  COLORS,
+} from './data';
```

Comment:
COLORS should be places at the common place like `/utils` or `/themes`.
All data should be predefined in `Charts.data.ts` file. not just `data.ts` file (rename it)